src/main/java/com/matchus/global/jwt/JwtAuthenticationFilter.java

```diff
@@ -57,7 +57,9 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
 					if (isNotEmpty(username) && authorities.size() > 0) {
 						JwtAuthenticationToken authentication =
 							new JwtAuthenticationToken(
-								new JwtAuthentication(token, username), null, authorities);
+								new JwtAuthentication(token, username), null,
+								authorities
+							);
```

Comment:
```suggestion
								new JwtAuthentication(token, username), 
								null,
								authorities
							);
```
아예 구분하려고 한다면 같이 구분해주는게 좋을 것 같아요!